src/main/webapp/i18n/en/tutorialGroups.json

```diff
@@ -0,0 +1,86 @@
+{
+    "artemisApp": {
+        "tutorialGroup": {
+            "deleted": "Tutorial group deleted"
+        },
+        "tutorialGroups": {
+            "tutorialGroupsButton": "Tutorial Groups"
+        },
+        "manageTutorialGroups": {
+            "title": "Tutorial Groups Management",
+            "tableCaption": "Tutorial Groups defined for this course",
+            "createButton": "Create Tutorial Group",
+            "columns": {
+                "id": "ID",
+                "title": "Title",
+                "teachingAssistant": "Teaching Assistant",
+                "capacity": "Capacity",
+                "registeredStudents": "Registered Students",
+                "isOnline": "Online/Offline",
+                "online": "Online",
+                "offline": "Offline",
+                "location": "Location",
+                "language": "Language",
+                "additionalInfo": "Additional Information"
+            },
+            "deleteButton": {
+                "question": "Are you sure to delete the Tutorial Group with title {{ title }}?",
+                "typeNameToConfirm": "Please enter the title of the tutorial group to confirm."
+            }
+        },
+        "createTutorialGroup": {
+            "title": "Create Tutorial Group"
+        },
+        "editTutorialGroup": {
+            "title": "Edit Tutorial Group"
+        },
+        "registeredStudents": {
+            "title": "Registered Students"
+        },
+        "tutorialGroupForm": {
+            "titleInput": {
+                "label": "Title",
+                "placeholder": "Group 1: Monday 12:00 - 13:00",
```

Comment:
```suggestion
                "placeholder": "Group 1: Monday {{from}} - {{to}}",
```

Reply:
But this is just the placeholder?

Reply:
Done in 82a5edb5cffebb59da7db909205529cf1416d77b